### PR TITLE
E2E tests

### PR DESCRIPTION
**Pull request by stevenvachon**
See the commit messages for the breakdown.

### CODE REVIEW

File: cypress/integration/temp.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import { e2e } from '@grafana/e2e';
+
+export interface DataSourceConfig {
```
Comment (ryantxu) [inline cypress/integration/temp.ts]: Maybe a differnet name... this is easy to confuse with the actual DataSourceConfig

Comment (stevenvachon) [reply cypress/integration/temp.ts]: Hmm, maybe `E2EDataSourceConfig`. Which other `DataSourceConfig` are you referring to?

Comment (ryantxu) [reply cypress/integration/temp.ts]: Datasouce config typically refers to the config for the datasouce -- in this case what we would be typing in the config page.  Not sure the best name here -- submit config?  SaveConfig?

Comment (stevenvachon) [reply cypress/integration/temp.ts]: It's a config for the `addDataSource()` function, which navigates pages and fills out forms. It does more than interact with the edit data source page.

Comment (ryantxu) [reply cypress/integration/temp.ts]: 👍 for `AddDataSourceConfig` -- this only stuck out because there is already so much confusion on DataSourceConfig

**Pull request merged**